src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+
+  transition-duration: $transition-time;
```

Comment:
https://github.com/mate-academy/layout_catalog/pull/4066#discussion_r1478283752 - this comment was about the approach of setting the animation speed, when we set it for all styling elements, we add an extra load, it should be set only for those parameters that we expect to use.
Fix this issue in all styles.